packages/mgt-chat/src/components/ChatList/ChatList.tsx

```diff
@@ -214,7 +236,7 @@ export const ChatList = ({
   }, []);
 
   const onClickChatListItem = (chat: GraphChatThread) => {
-    chatListClient?.setInternalSelectedChat(chat);
+    chatListClient?.setSelectedChat(chat);
   };
 
   const chatListButtonItems = props.buttonItems === undefined ? [] : props.buttonItems;
```

Comment:
line 277 doesn't include chatListClient as a dependency. Is there a reason?

Reply:
added